.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -14,7 +14,7 @@ body:
       label: Is there an existing issue outlining your problem?
       description: Please search to see if an issue already exists for your problem.
       options:
-        - label: I have searched the existing issues and they do not solve my problem.
+        - label: I have searched the existing issues and troubleshooting guide they do not solve my problem.
```

Comment:
Is it possible to insert a markdown link for the text "troubleshooting guide" ?

Reply:
Done